src/main/java/uk/gov/di/ipv/lambda/CredentialIssuerHandler.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.di.ipv.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.oauth2.sdk.util.StringUtils;
+import uk.gov.di.ipv.domain.ErrorResponse;
+import uk.gov.di.ipv.helpers.ApiGatewayResponseGenerator;
+import uk.gov.di.ipv.helpers.RequestHelper;
+import uk.gov.di.ipv.helpers.ResponseBodyHelper;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+public class CredentialIssuerHandler implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    private Set<String> validCredentialIssuers = Set.of("PassportIssuer", "FraudIssuer");
+
+    @Override
+    public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent input, Context context) {
+
+        Map<String, String> body = ResponseBodyHelper.parseRequestBody(input.getBody());
```

Comment:
ResponseBodyHelper.parseRequestBody() ??? 🤯

Reply:
we renamed to RequestBodyHelper